server/tests/message.integration.spec.js

```diff
@@ -29,16 +29,20 @@ const testMessageObject = {
 const testMessageArray = [];
 const fromArray = ['user0', 'user1', 'user2', 'user3'];
 const MessageUnscoped = Message.unscoped();
+let date = new Date();
 // 4 senders send message to 4 recipients each
 fromArray.forEach((receiver) => {
     fromArray.forEach((sender, index) => {
+        date = new Date(date);
```

Comment:
What's the idea behind (1) recreating and (2) setting seconds +1?

Reply:
2: to generate a distinct `createdAt` time for each message to test the ordering
1: if we didn't create a new date object each time, it would be the same reference in each object in the bulk create and they would end up being the same time. I think we could `.toString()` here to avoid creating a new date object, but it's only 16 objects